app/views/api/v2/os_default_templates/base.json.rabl

```diff
@@ -1,3 +1,7 @@
 object @os_default_template
 
-attributes :id, :config_template_id, :config_template_name, :template_kind_id, :template_kind_name
+attributes :id => :id,
+           :provisioning_template_id => :config_template_id,
```

Comment:
Should (can?) we have both here?  I wonder if always sticking to config_template_id might make migrating harder than listing both.